packages/v4/src/content/design-guidelines/usage-and-behavior/filters/filters.md

```diff
@@ -12,210 +12,315 @@ related: [
 
 #### Behavior
-Filter queries are applied when the user selects or deselects an item in the list, or types a query in the input field and presses Enter.
+A filter query is triggered whenever a new value is set by the user. The following are triggers applicable to the variations of the type-ahead filter shown in the previous example.
 
-## Usage
-### Layout considerations
-Filters may be included in a toolbar and are usually left aligned. There may be more than one filter used in a single toolbar, and in some cases, custom filters may make up a full toolbar. Filters may also be used alone outside of a toolbar.
+| Value selector | Trigger |
+| ---------------- | ------ |
+| Search field | The user types in an input and/or presses the Enter key after typing in the input field. This action will create a filter for the first value shown in the filtered list. |
+| Select lists | The user selects or deselects an item in the list. |
 
-### Mobile considerations
-In responsive views, filters may be hidden within the filter icon. On click, the toolbar will expand to reveal the filter. If there are multiple filter elements, they will be collapsed into a single faceted filter.
 
-<img src="./img/mobile.png" alt="Mobile" />
+## Layout considerations
+Filters may be included in a [toolbar](/components/toolbar) and are usually left aligned. There may be more than one filter used in a single toolbar, and in some cases, custom filters may make up a full toolbar. Filters may also be used alone outside of a toolbar.
 
+## Mobile considerations
+In responsive views, filters may be hidden within the filter icon. On click, the toolbar will expand to reveal the filter. If there are multiple filter elements, they will be collapsed into a single faceted filter. Read more about toolbars in mobile views in [the toolbar design guidelines](/components/toolbar/design-guidelines#the-toolbar-on-mobile).
 
-## Examples
-**Filter in toolbar on data list**
-<img src="./img/example1.png" alt="Example1" />
+<img src="./img/mobile.png" alt="mobile view" width="880" />
 
-**Multiple filters in toolbar**
-<img src="./img/example2.png" alt="Example2" />
+### Examples
+**Filter group in data list toolbar**
+<img src="./img/ex-datalist.png" alt="example of filter group in data list" width="880" />
 
 
+**Attribute filter in table toolbar**
+<img src="./img/ex-table.png" alt="example of attribute value filter in table " width="880" />
+
 ## Content
 
 ### Filters
-Filter hint text in input fields should read “Filter by [attribute]...” and be in sentence case.
+Filter placeholder text in input fields should read “Filter by [attribute name].” Write all placeholder text in sentence case.
+
+**Examples:** _Filter by status_, _Filter by vendor_
 
 ### Clearing filters
```

Comment:
I remember some discussion about using "Reset filters" instead of clear in certain use cases. Should we include that here or would you rather leave out of PatternFly?

Reply:
I feel like I would probably leave that out of PatternFly for now!